templates/bitbucket/myrror-cli.bitbucket-pipelines.yml

```diff
@@ -0,0 +1,70 @@
+pipelines:
+  default:
+    - step:
+        name: Myrror Scan check for PR to YOUR_MAIN_BRANCH
```

Comment:
is YOUR_MAIN_BRANCH a token? need to use $YOUR_MAIN_BRANCH no?

Reply:
no, people have to change YOUR_MAIN_BRANCH to branch name (main as example)